ftplugin/lean4/switch.vim

```diff
@@ -0,0 +1,39 @@
+" This possibly belongs in lean.vim or switch.vim itself but putting it here
+" for now.
+
+if !exists("g:loaded_switch")
+  finish
+endif
+
+let b:switch_definitions = [
+    \ g:switch_builtins.true_false,
+    \ ["#check", "#eval", "#reduce"],
+    \ ["tidy", "suggest", "hint", "linarith", "library_search"],
+    \ ["rw", "simp", "simp?"],
+    \ ["cases", "rcases", "obtain"],
+    \ ["norm_cast", "push_cast"],
```

Comment:
At least these are lean3/mathlib-specfic.

Reply:
Gotcha, I just copied this from `ftplugin/lean/switch.vim`... I'm not personally familiar with the differences here yet, so there may be others.